### ISSUE TITLE
Move "share" view to new details view

### ISSUE THREAD

**Issue by k88hudson** [mozilla/webmaker-app#870]
Relies on https://github.com/mozilla/webmaker-app/issues/868, base design here: https://redpen.io/wy5af842444b6a2667

@vazquez do you have designs for this subview specifically yet or does it still need to be done?

**Comment by vazquez** [comment]
Needs to be done still Kate. I will make it a priority to unblock #868

**Comment by thisandagain** [comment]
We decided to punt moving re-designing the share view for now. Closing in favor of keeping the existing share view.
